pkg/task/restore_raw.go

```diff
@@ -0,0 +1,133 @@
+package task
```

Comment:
Please add license header.

Reply:
I found that other files in the project have no license headers either.

Reply:
License header added in #182 .